src/igvfd/audit/analysis_set.py

```diff
@@ -14,19 +14,19 @@ def audit_input_file_sets(value, system):
     '''
     [
         {
-            "audit_description": "Primary analysis sets are expected to have at least one measurement set as an input file set.",
```

Comment:
I think this audit is being removed entirely by IGVF-1675, so perhaps this ticket should wait a little bit and rebase?